tests/styles.spec.js

```diff
@@ -0,0 +1,55 @@
+const assert = require('assert');
+const { execSync } = require('child_process');
+const { mkdirSync, cpSync, readFileSync } = require('fs');
+
+/* eslint-disable */
+let copy;
+if (typeof cpSync === 'undefined') {
+  copy = require('recursive-copy');
+}
+/* eslint-enable */
+
```

Comment:
Why do we need this? 🤔

Reply:
Without it, the `lint-js` job flags it as an issue: https://github.com/ChromaticHQ/calliope/runs/5666351285?check_suite_focus=true

Reply:
I see, but what that error is saying is that `copy` is declared but never used. So I don't think we need these lines at all (from the `let copy` onwards). I think I may have added these lines by mistake, so let's remove them.
```suggestion
```

Reply:
To clarify: it is accurate that we need `copy` in lieu of `cpSync`, but that is handled in the `helpers.js` file. There should be no need for it here.

Reply:
Ahh I see now, sorry about that! This has been cleaned up now as well.